src/models/EventMapper.php

```diff
@@ -453,6 +453,8 @@ public function transformResults($results, $verbose)
                             . $row['ID'] . '/comments/reported';
                         $list[$key]['reported_talk_comments_uri'] = $base . '/' . $version . '/events/'
                             . $row['ID'] . '/talk_comments/reported';
+                        $list[$key]['pending_claims_uri'] = $base . '/' . $version . '/events/'
```

Comment:
Yes it is - see https://github.com/liam-wiltshire/joindin-api/blob/e9839cf7164e0388d8cb413959ddf9803ff399f4/src/models/EventMapper.php#L451